.github/workflows/docker-CI.yml

```diff
@@ -2,7 +2,7 @@ name: Docker Build and Push
 
 on:
   push:
-    branches: [ master, develop ]
+    branches: [ master, develop, ma_jovanovic ]
```

Comment:
Can be removed right?

Reply:
True

Reply:
About the blockchain verification: it is now being done by the PCS, so no need for the old system.

Reply:
There is a new check now: usesPrivacyControl